core/modules/wmgr/test/wmgrDbTest.py

```diff
@@ -0,0 +1,228 @@
+#!/bin/env python
+#
+# LSST Data Management System
+# Copyright 2015 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+"""Test application for wmgr database management commands.
+
+This is a unit test but it requires few other things to be present:
+- mysql server up and running
+- zookeeper up and running
+- application config file, passed via WMGRCONFIG envvar
+
+@author Andy Salnikov - salnikov@slac.stanford.edu
+"""
+
+#------------------------------
+#  Module's version from CVS --
+#------------------------------
+__version__ = "$Revision: 8 $"
+# $Source$
+
+#--------------------------------
+#  Imports of standard modules --
+#--------------------------------
+import logging
+import sys
+import unittest
+
+#-----------------------------
+# Imports for other modules --
+#-----------------------------
+from flask import Flask, json
+from lsst.qserv.wmgr.config import Config
+from lsst.qserv.wmgr import dbMgr
+
+#---------------------
+# Local definitions --
+#---------------------
+
+#-------------------------------
+#  Unit test class definition --
+#-------------------------------
+
+logging.basicConfig(level=logging.WARNING)
+
+
+class wmgrDbTest(unittest.TestCase):
```

Comment:
Nice example. But is it possible to generate a doc for each of the url this service provide? It would help to use it.